docs/src/app_dev/app_val_tokenomics/index.rst

```diff
@@ -0,0 +1,121 @@
+This is followed by
+a *minting phase* where the owners of an activity record can mint CC proportional to its minting weight. There are several rounds active
+concurrently with each round being in a different phase. A round starts
+every 10 minutes, which is configuration parameter that the Super Validators may change in the future via a governance vote. See the CC
+whitepaper for the details.
+
+There are five key templates involved in the accounting for network activity:
+
+-  Two templates are application related:
+
+      - :ref:`FeaturedAppActivityMarker <type-splice-amulet-featuredappactivitymarker-16451>`
+
+      - :ref:`AppRewardCoupon <type-splice-amulet-apprewardcoupon-57229>`
+
+-  Three templates relate to providing the infrastructure for the activity:
+
+      - :ref:`ValidatorRewardCoupon <type-splice-amulet-validatorrewardcoupon-76808>`
+
+      - :ref:`ValidatorLivenessActivityRecord <type-splice-validatorlicense-validatorlivenessactivityrecord-17293>`
+
+      - :ref:`SvRewardCoupon <type-splice-amulet-svrewardcoupon-68580>`
+
+The last four are activity records while an ``FeaturedAppActivityMarker`` is not considered an activity record. As discussed later, a
+``FeaturedAppActivityMarker`` is converted into an ``AppRewardCoupon`` via
+automation.
+
+The ``FeaturedAppActivityMarker``,
+``AppRewardCoupon``, and ``ValidatorRewardCoupon`` contracts are created when an
+application's transaction succeeds. In general, an application receives rewards when its Daml code directly creates ``FeaturedAppActivityMarker`` contracts
+or interacts with Daml models that feature the application provider's party.
+
+As an aside, other templates that are important to the tokenomics are:
+
+- :ref:`Amulet <type-splice-amulet-amulet-63582>`, which stores the fees schedules;
+
+- :ref:`OpenMiningRound <type-splice-round-openmininground-90060>`, which stores the price and fees as of opening the round;
+
+- :ref:`IssuingMiningRound <type-splice-round-issuingmininground-98097>`, which stores the amount-to-mint-per-activity-right.
+
+There is no difference in activity record creation for an `external
+party <https://docs.digitalasset.com/build/3.3/tutorials/app-dev/external_signing_onboarding.html#tutorial-onboard-external-party>`__
+or local party but there is a difference in the automation support used
+in the minting phase. For local parties onboarded to a validator, the
+validator application runs background automation to mint all activity
+records automatically. An external party signs transactions using a key
+they control. As a consequence, the validator automation is not able to
+perform minting for external parties. For external parties, automation needs
+to be developed to call ``AmuletRules_Transfer`` at least once per round
+with all activity records as inputs.  A draft CIP, called "Weighted Validator Liveness Rewards for SV-Determined Parties",
+will be available for comment soon which describes providing this support.
+cip-0073/cip-0073.md
```

Comment:
please link to the mailing list topic on cip-discuss for it, so people can learn more about it.

Reply:
Done